### PR TITLE
Fix for issue #31

### PR DESCRIPTION
**Pull request by markxroberts**
This code change fixes issue #31.  I haven't found any issues with this yet.  For the bypass switches the alternative fix is to set the 'optimistic' flag to 'true'.

### CODE REVIEW

**Comment by vanackej** [comment]
I did it this way originally, but wasn't happy with it because it reported the previous known state even if it was really old and sometimes inaccurate. For an alarm system, I don't think it is a desired behavior.
Wouldn't it be better to subscribe to home assistant availability topic, and send the current sensors state again on HA availability publication ? What do you think about it ?

**Comment by markxroberts** [comment]
I see the problem.  I haven’t experienced these issues myself.  When might this occur in reality?
I find it more frustrating when the system reports an unknown state for all sensors at restart.
The solution you describe is the most elegant!

**Comment by markxroberts** [comment]
Update to fix unavailability status following Home Assistant restart

**Comment by ob1w4nken0b1** [comment]
Wow! Can't wait to test this

File: README.md [hunk 1]
```diff
@@ -99,6 +101,8 @@ risco-mqtt-local supports [mqtt auto discovery](https://www.home-assistant.io/do
 
 Default `<discovery_prefix>` is **homeassistant**. You can change it by overwriting the value within **home-assistant-discovery-prefix** config.
 
+Home assistant auto discovery republizhed on Home Assistant restart.
```
Comment (vanackej) [inline README.md:104]: Small typo : republished

**Comment by vanackej** [comment]
Looks good.
I'm not at home this week, I'll be able to test and merge next week. 
Thanks for the contribution 👍

**Comment by markxroberts** [comment]
I’ve shared the Javascript file directly with @ob1w4nken0b1
If you know how/where to replace this in your setup, you will be able to test.

**Pull request merged**